test/js_optimizer/unsignPointers-output.js

```diff
@@ -37,3 +37,6 @@ heap[x >>> 0];
 HEAP[x >>> 0];
 
 HeAp[x];
+
+// but not this
```

Comment:
Please move this comment in the test so it is before the code, which should make it remain in the right place, I hope.